lib/bundler/cli/gem.rb

```diff
@@ -139,6 +139,10 @@ def run
 
       # Open gemspec in editor
       open_editor(options["edit"], target.join("#{name}.gemspec")) if options[:edit]
+    rescue SystemCallError => e
```

Comment:
ah in that case, please use both `filesystem_access` and separately rescue `EEXIST` and raise a `GenericSystemCallError` for it?